### PR TITLE
Use classic hero icons instead of reforged

### PR DESCRIPTION
**Pull request by francislavoie**
Copied the classic heroes from `public/assets/raceAvatars/classic/` and renamed them to match the hero names instead of the `RACE_#.jpg` naming

### CODE REVIEW

File: src/components/overall-statistics/HeroPictureSelect.vue [hunk 1]
```diff
@@ -156,7 +156,7 @@ export default defineComponent({
     }
 
     function parsePicture(hero: HeroPick): string {
-      return getAsset(`heroes/${hero.heroId}.png`);
```
Comment (marcoabreu) [inline src/components/overall-statistics/HeroPictureSelect.vue:159]: Shall we maybe centralized this function so we don't have this logic replicated in multiple places?

Comment (francislavoie) [reply src/components/overall-statistics/HeroPictureSelect.vue:159]: I'm not against it, but maybe in a separate PR. I'm still new here :sweat_smile:

**Comment by gustav87** [comment]
I believe the images in `public\assets\raceAvatars\classic` are actually pngs, with the wrong extension. Shall we set the correct .png extension and use that instead?

**Comment by francislavoie** [comment]
Oh you're right, hilarious :joy: that's easier

**Comment by gustav87** [comment]
image

**Comment by francislavoie** [comment]
Done, now it's just updated files, no add/delete, no code changes

**Review by gustav87 — APPROVED** [review]

**Pull request merged**